content/applications/websites/website/web_design/building_blocks/features_blocks.rst

```diff
@@ -0,0 +1,28 @@
+===============
+Features blocks
+===============
+
+Use the :guilabel:`Features` blocks to add advanced functionalities to your website, such as
+describing the attributes of a product or service. The blocks from this category allow you to
+present the content of your website in an illustrative method rather than sticking to the textual
+content. This documentation presents the :guilabel:`Table of content` building block.
+
+.. image::features_blocks/features_blocks.png
+   :alt: The features blocks available in the website builder.
+
+.. seealso::
+   :doc:`Building blocks <../building_blocks>`
+
+Table of content
+================
+
+You can add a :guilabel:`Table of content` building block to your website by dragging and dropping
+it from the :guilabel:`Features` blocks section in the website configurator.
+
+Once added, you can customize the appearance and functionality of the table of contents.
+
+Header 2 titles automatically appear on the left of your page, helping users navigate your content
+more efficiently.
+
+.. image:: features_blocks/table_of_content.png
+   :alt: Displaying the table of content building block
```

Comment:
do you think we need to see the whole screen, maybe the edit options on the right are not useful, not sure, but I feel like it's not focusing on the H titles on the left